tests/Eccube/Tests/Web/Admin/Product/ProductContorllerTest.php

```diff
@@ -48,7 +49,7 @@ public function createFormData()
             'description_detail' => $faker->text,
             'description_list' => $faker->paragraph,
             'Category' => null,
-            'Tag' => 1,
+            'Tag' => '',
```

Comment:
Dear @chihiro-adachi san,
This was my mistake when editing the tag to run the test.
I will fix it immediately.
Thanks,
Dung Le